.travis.yml

```
@@ -0,0 +1,40 @@
+language: cpp
+notifications:
+  email: false
+sudo: required
+before_script:
+  # 
+  - wget "https://www.shoup.net/ntl/ntl-11.4.3.tar.gz"
+  - tar xf ntl-11.4.3.tar.gz
```

```suggestion
  - tar --no-same-owner -xf ntl-11.4.3.tar.gz
```

`--no-same-owner` is required to avoid a potential back door.